Source/WebInspectorUI/UserInterface/Views/ConsoleDrawer.js

```diff
@@ -153,8 +153,8 @@ WI.ConsoleDrawer = class ConsoleDrawer extends WI.ContentBrowser
 
     _updateDrawerHeight(height)
     {
-        const minimumHeight = 64;
-        const maximumHeight = this.element.parentNode.offsetHeight - 100;
+        let minimumHeight = 64;
```

Comment:
Style: this could still be `const`

Reply:
Ha! I changed them to `let` so I can modify their values with actions in breakpoints in Web Inspector, then forgot to return them to `const`. Fixed now :)